UPGRADE.md

```diff
@@ -1,3 +1,39 @@
+# Ilios 3 Upgrade Notes
+
+## General steps
+
```

Comment:
It might be helpful to mention that, while we use 'sudo -u apache' in our examples, this command should be run in the context of the user that runs web services (apache, www, etc)...